src/blend.js

```diff
@@ -0,0 +1,19 @@
+var foldl = require('./foldl');
+var mixin = require('./mixin');
+
+
+/**
+ * Merges a list of objects together into one object.
+ *
+ * @func
+ * @memberOf R
+ * @category List
+ * @sig [{k: v}] -> {k: v}
+ * @param {Array} list An array of objects
+ * @return {Object} A merged object.
+ * @see foldl
+ * @example
+ *
+ *      R.blend([{foo:1},{bar:2},{baz:3}]); //=> {foo:1,bar:2,baz:3}
```

Comment:
It would be nice to demonstrate whether left trumps right or vice versa by having two objects share a key.